demo/src/index.js

```diff
@@ -44,69 +44,71 @@ class App extends React.Component {
   }
 
   render() {
-    return <div className="App">
-      <h1>
-        <code>&lt;<a href="https://github.com/insin/react-maskedinput">MaskedInput</a>/&gt;</code>
-      </h1>
-      <p className="lead">A React component which creates a masked <code>&lt;input/&gt;</code></p>
-      <div className="form-field">
-        <label htmlFor="card">Card Number:</label>
-        <MaskedInput mask="1111 1111 1111 1111" name="card" id="card" size="20" value={this.state.card} onChange={this._onChange}/>
+    return (<React.StrictMode>
```

Comment:
why add this? I think it'd better for the demo to not have an arbitrary performance block on it

Reply:
I don;t understand how StrictMode is a performance block, care to explain?